packages/@expo/fingerprint/src/__tests__/Fingerprint-test.ts

```diff
@@ -372,3 +372,46 @@ describe(diffFingerprints, () => {
     ]);
   });
 });
+
+describe('function api stability', () => {
+  const Fingerprint = require('../index');
+  afterEach(() => {
+    vol.reset();
+  });
+
+  it('maintains consistent hash and function signature expected by eas-cli', async () => {
```

Comment:
```suggestion
  it('maintains consistent function signature and returned structure expected by eas-cli', async () => {
```

nit: remove consistent hash